content/css/concepts/inheritance/inheritance.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'Inheritance'
+Description: 'The cascading in CSS. Styling that is applied to a chilled element when they are applied to its parent.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Inheritance'
+  - 'Style'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+CSS inheritance is the cascading in CSS. Inheritance is when styling that is applied to a chilled element when they are applied to its parent. Some properties are inherited but others are not.
+
+## Inherided propities
+
+<!-- prettier-ignore -->
+| Property |
+| --- |
+| [`color`](https://www.codecademy.com/resources/docs/css/colors) |
+| [`visibility`](https://www.codecademy.com/resources/docs/css/visibility) |
+| All [Typograpgy](https://www.codecademy.com/resources/docs/css/typography) properties exept `text-decoration` and `text-overflow`. |
+| [`cursor`](https://www.codecademy.com/resources/docs/css/cursor) |
+
+## Non inherided propities
+
+<!-- prettier-ignore -->
+| Property |
+| --- |
+| All [`background` properties](https://www.codecademy.com/resources/docs/css/background) |
```

Comment:
```suggestion
| All [background](https://www.codecademy.com/resources/docs/css/background) properties | Adds wallpaper visuals for HTML elements. | 
```
I would remove the backticks since this is linking to the CSS background concept and not the actual `background` property term entry. This suggestion also applies to some more rows down below.